mkosi/__init__.py

```diff
@@ -300,7 +300,8 @@ def configure_extension_release(context: Context) -> None:
             f.write(f"{prefix}_VERSION_ID={context.config.image_version}\n")
 
         if f"{prefix}_SCOPE" not in extrelease:
-            f.write(f"{prefix}_SCOPE=initrd system portable\n")
+            if scope := context.config.environment.get(f"{prefix}_SCOPE", "initrd system portable"):
```

Comment:
Why guard this behind an if? The only case this condition can be false is if `{prefix}_SCOPE` is defined but empty.  If this is intended, than a comment would be good here, to say that this is indeed a case that's been considered and we want to allow people to explicitly unset this, otherwise it might make sense to write the empty value and leave the using software to deal with this.

Reply:
Was a leftover from before I added the default back